### ISSUE TITLE
Build error with `using Statistics` in startup.jl

### ISSUE THREAD

**Issue by antoine-levitt** [JuliaLinearAlgebra/Arpack.jl#59]
I have `using Statistics` in my ~/.julia/config/startup.jl. With that in, ] build Arpack fails with 

```
┌ Error: Error building `Arpack`: 
│ ERROR: LoadError: ArgumentError: Package Statistics not found in current path:
│ - Run `Pkg.add("Statistics")` to install the Statistics package.

```
If I remove the `using Statistics`, everything is fine.

**Comment by andreasnoack** [comment]
@KristofferC Any idea what could be going on here?

**Comment by fredrikekre** [comment]
Julia version? This issue has been fixed in 1.0.2 IIRC.

You can start julia with `--startup-file=no` and rebuild.

**Comment by antoine-levitt** [comment]
This is 1.0.0; closing, thanks!

**Comment by antoine-levitt** [comment]
(I tested, it's fixed on 1.1)
